### ISSUE TITLE
multiple node filenames?

### ISSUE THREAD

**Issue by ladydemona** [jexp/batch-import#87]
is there a binary build of this that has the importing of multiple nodes files (i.e filenames concatenated without spaces) that works?  I've been trying it but it seems to only import the first node file.

**Comment by cescalante-carecloud** [comment]
so doing something like this doesn't work for you?
nodefile1.csv,nodefile2.csv,nodefile3.csv relsfile1.csv,reslfile2.csv

What happens when you have one node file and one relationship file?
nodefile1.csv relfile1.csv

**Comment by ladydemona** [comment]
That's correct, it appeared to only import only the first csv that was listed-- I also tried both in the properties file and command line.  It imports fine with just 1 :)

**Comment by cescalante-carecloud** [comment]
can you paste your import.sh file (so i can try to recreate). 
Was this working for you before? I just ran into a mystery issue today that I am not yet sure how to replicate.

**Comment by ladydemona** [comment]
HEAP=4G
DB=${1-target/graph.db}
shift
shift
NODES=${1-nodes.csv}
shift
RELS=${1-rels.csv}
CP=""
for i in lib/*.jar; do CP="$CP":"$i"; done
#echo java -classpath $CP -Xmx$HEAP -Xms$HEAP -Dfile.encoding=UTF-8
org.neo4j.batchimport.Importer batch.properties "$DB" "$NODES" "$RELS" "$@"
java -classpath $CP -Xmx$HEAP -Xms$HEAP -Dfile.encoding=UTF-8
org.neo4j.batchimport.Importer batch.properties "$DB" "$NODES" "$RELS" "$@"

On Wed, Feb 12, 2014 at 3:21 PM, cescalante-carecloud <
notifications@github.com> wrote:

> can you paste your import.sh file
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/jexp/batch-import/issues/87#issuecomment-34912922
> .

## 

-C
